include/freerdp/utils/passphrase.h

```diff
@@ -31,9 +31,46 @@ extern "C"
 {
 #endif
 
-	FREERDP_API int freerdp_interruptible_getc(rdpContext* context, FILE* file);
+	/** @brief Read a single character from a stream.
```

Comment:
As these are new functions, we should have a `@since` tag

Reply:
no, not new, just did not have documentation.
the `suggest` stuff is new though.